docs/app/pages/components/components-sections/input-controls/input-dropdown/input-dropdown.component.html

```diff
@@ -105,6 +115,9 @@
     <tr uxd-api-property name="selectedChange" type="T[]">
         Emits when the selection changes. The event will contain the new selected items.
     </tr>
+    <tr uxd-api-property name="dropdownOpenChange" type="boolean">
+        Emits when <code>dropdownOpen</code> changes.
```

Comment:
Should perhaps be more explanatory like our tooltip docs, eg:

Emits when the dropdown is shown or hidden. The event will contain a boolean representing the visibility.